packages/plans-grid/src/plans-interval-toggle/index.tsx

```diff
@@ -57,20 +57,24 @@ const PlansIntervalToggle: React.FunctionComponent< PlansIntervalToggleProps > =
 	maxMonthlyDiscountPercentage,
 	className = '',
 } ) => {
-	const { __, hasTranslation } = useI18n();
+	const { __, _x, hasTranslation } = useI18n();
 	const locale = useLocale();
 
 	const fallbackMonthlyLabel = __( 'Pay monthly', __i18n_text_domain__ );
 	// Translators: intended as "pay monthly", as opposed to "pay annually"
-	const newMonthlyLabel = __( 'Monthly', __i18n_text_domain__ );
+	const newMonthlyLabel = _x( 'Monthly', 'Adverb (as in "Pay monthly")', __i18n_text_domain__ );
 	const monthlyLabel =
-		locale === 'en' || hasTranslation?.( 'Monthly' ) ? newMonthlyLabel : fallbackMonthlyLabel;
+		locale === 'en' || hasTranslation( 'Monthly', 'Adverb (as in "Pay monthly")' )
```

Comment:
Sorry to be mentioning it after the PR got merged, but now that we're using `@wordpress/react-i18n`, `__i18n_text_domain__` can be passed as third argument to `hasTranslations`.

Reply:
No worries. When should it be passed?

Reply:
Thank you for letting us know! It _can_ be passed or it _should_ be passed? Do we need to open a new PR where we always pass the context?

Reply:
When not passed, `hasTranslation` would be looking into the `default` text domain. Therefore, it should be passed when you expect `__i18n_text_domain__` to be something other than `default`.
AFAIK, this package is currently being used in Gutenboarding and ETK. So for Gutenboarding, it should be fine, as all translations are being loaded on the default text domain, but on the other hand ETK uses the `full-site-editing` text domain, so there it will be looking into the wrong text domain.

Reply:
Thanks for elaborating! I'm creating an issue + PR for a fix.